scss/_pagination.scss

```
@@ -33,12 +33,19 @@
   &:first-child {
     .page-link {
       margin-left: 0;
-      @include border-left-radius($border-radius);
+      @include border-left-radius($pagination-border-radius);
     }
   }
   &:last-child {
     .page-link {
-      @include border-right-radius($border-radius);
+      @include border-right-radius($pagination-border-radius);
+    }
+  }
+
+  //Add border-radius to all pageLinks in case they have left margin
+  @if $pagination-margin-left != ( - $pagination-border-width ) {
```

We can use a if/else statement here to prevent the generation of the `:last-child` when ` $pagination-margin-left != ( - $pagination-border-width )`

This line can also be simplified as 
```suggestion
  @if $pagination-margin-left != -$pagination-border-width {
```